tests/baselines/reference/jsdocTypeReferenceToValue.types

```diff
@@ -1,7 +1,7 @@
 === tests/cases/conformance/jsdoc/foo.js ===
 /** @param {Image} image */
 function process(image) {
->process : (image: Image) => HTMLImageElement
+>process : (image: new (width?: number, height?: number) => HTMLImageElement) => HTMLImageElement

```

Comment:
👀

Reply:
Yeah, `Image` is a constructor with no associated `interface`, and jsdoc lookup is strange. So, in TS, `(image: Image) => HTMLImageElement` will issue a `Image is not a type` error (because it's not). So we gotta serialize the type the jsdoc resolver resolved to instead of keeping the input node.